src/server.c

```diff
@@ -4830,6 +4837,7 @@ sds genRedisInfoString(const char *section) {
             "mem_fragmentation_bytes:%zd\r\n"
             "mem_not_counted_for_evict:%zu\r\n"
             "mem_replication_backlog:%zu\r\n"
+            "mem_replication_buffer:%zu\r\n"
```

Comment:
first, i think we should call them "buffers" (not "buffer").
first because it's a linked list of allocations (same as we call the other ones "client output buffers"), but secondly, because they serve multiple purposes (multiple replicas, and also the backlog)

secondly, maybe we should use the word "total"? i.e. in the past the `mem_clients_slaves` and `mem_replication_backlog` were each representing a portion of the memory, and you had to sum them to get the total.
now they represent overlapping memory, and this metric is the actual used total.
```suggestion
            "mem_total_replication_buffers:%zu\r\n"
```

what do you think?